### PR TITLE
Added GUI to create short URLs

### LINKED ISSUE #2: UI design for the front-end and develop it.
The `API` can be called by other programs and can be used independently but the UX is not good.
Propose a `UI` design for the front-end in the comments and we will develop it. If any doubts, leave a comment mentioning me.

### PR DESCRIPTION
**Pull request by RishiRaj22**
Fixes #2

#### Checklist

- [ ] I have read the CONTRIBUTING guidelines and my PR follows them.
- [ ] My branch is up-to-date with the Upstream `master` branch.
- [ ] The unit tests pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works 
- [ ] I have added necessary documentation (if appropriate)

- [ ] All the functions created/modified in this PR contain relevant docstrings.

#### Short description of what this resolves:
Added a basic GUI for the web app using jQuery.

#### Changes proposed in this pull request:

- The new user interface is responsive
- It is easy to use
- Short URL can be copied by a single press of a button

### CODE REVIEW

File: server/public/script.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+$('#input_form').submit(function() {
+    const val = $('#input_link').val();
+    console.log('val',val);
```
Comment (zamhaq) [inline server/public/script.js]: remove the console log

File: server/public/script.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    $.get('/new/'+val,function(data) {
+        console.log('data',data);
```
Comment (zamhaq) [inline server/public/script.js]: here also

File: server/views/index.html [hunk 3]
```diff
@@ -1,23 +1,30 @@
...
+    <div class = "form_div" id = "input_div">
+        <h2>Enter a link and submit it to get a short version</h2>
+        <form id = "input_form">
+            <input id="input_link" placeholder="https://www.google.com">
+            <input id = "submit_link" type="submit">
+        </form>
+    </div>
+    <div class = "form_div" id="output">
+        <h2>Shortened link</h2>
+        <input id="output_link" value = "http://safe-crag-80479.herokuapp.com/" readonly >
```
Comment (zamhaq) [inline server/views/index.html]: can you explain why you have set `value` to production URL?

**Comment by RishiRaj22** [comment]
The latest commit removes the logging and the dummy placeholder value. @zamhaq

**Comment by zamhaq** [comment]
lgtm @RishiRaj22 thanks

**Comment by RishiRaj22** [comment]
The PR is still not showing as merged. What is the meaning of approved PR? @zamhaq

**Comment by Logan1x** [comment]
@RishiRaj22 wait till he comes online he'll surely add your PR or he might need some changes from you, till then work on other issue's or make this pr better😊

**Pull request merged**